cluster/gce/gci/configure-helper.sh

```diff
@@ -341,6 +355,15 @@ function start-kubelet {
       flags+=" --hairpin-mode=${HAIRPIN_MODE}"
     fi
   fi
+  # Network plugin
+  if [[ -n "${NETWORK_PROVIDER:-}" ]]; then
+    flags+=" --network-plugin-dir=/home/kubernetes/bin"
```

Comment:
Note that for CNI this parameter sets the directory where _config_ is searched, not binaries.  As far as I can tell CNI just doesn't work on GCI now.